src/control/system/member.go

```diff
@@ -45,41 +45,57 @@ type MemberState int
 
 const (
 	// MemberStateUnknown is the default invalid state.
```

Comment:
If it doesn't violate a style rule, I would have preferred to see these values in hex, given that they are used as bit masks.  Just seems more natural.

Reply:
Done.